apps/demos/Demos/Stepper/Overview/jQuery/index.js

```diff
@@ -0,0 +1,76 @@
+$(() => {
+  const initialStepperSettings = {
+    rtlEnabled: false,
```

Comment:
rtlEnabled options has false as default value, it's not oblige declare its on initialization